BFWControls/Modules/Blur/View/BlurView.swift

```diff
@@ -19,21 +19,18 @@ class BlurView: UIView {
     //MARK: Accessors
 
     var blurredImage: UIImage? {
-        var blurredImage: UIImage? = nil;
-        if let subviews = superview?.subviews,
-            let selfIndex = subviews.index(of: self) {
-            if selfIndex > 0 {
-                let previousView = subviews[selfIndex - 1];
-                let contentImage = UIImage(of: previousView,
-                                           size: bounds.size)
-                blurredImage = UIImageEffects.imageByApplyingBlur(to: contentImage,
-                                                                  withRadius: blurRadius,
-                                                                  tintColor: backgroundColor,
-                                                                  saturationDeltaFactor: 1.8,
-                                                                  maskImage: nil)
-            }
-        }
-        return blurredImage
+        guard let subviews = superview?.subviews,
+            let previousIndex = subviews.index(of: self).map({ $0 - 1 }),
+            subviews.indices.contains(previousIndex)
```

Comment:
We don't actually want to know if previousIndex is a non-negative, what we want to know is if it's a valid index. Changing the code to betray this intention aids comprehension. In an abstract way this is similar to [the XY problem](https://meta.stackexchange.com/questions/66377/what-is-the-xy-problem/66378#66378).